samples/tfm/tfm_psa_test/sample.yaml

```diff
@@ -0,0 +1,24 @@
+common:
+    tags: tfm
+    platform_allow: nrf5340dk_nrf5340_cpuappns nrf9160dk_nrf9160ns
+    harness: console
+    harness_config:
+      type: multi_line
+      regex:
+        - "\\*\\*\\*\\*\\* PSA Architecture Test Suite - Version 1.0 \\*\\*\\*\\*\\*"
+        - "TOTAL FAILED    : 0"
+sample:
+  name: "TFM PSA Test"
+
+tests:
+  sample.tfm.psa_storage_test:
```

Comment:
From a testing perspective I prefer splitting the storage-suite into the PROTECTED_STORAGE and INTERNAL_TRUSTED_STORAGE suites - that way we get the results for the two suites separate instead of having them in the same run.